rep/processors/BaseProcessor.py

```diff
@@ -1,6 +1,11 @@
 from abc import ABC, abstractmethod
+import sqlite3
 
 class BaseProcessor(ABC):
+
+    def __init__(self):
```

Comment:
IMO, this class is doing too many things now. Imagine, for a second, that we add 5 more processors. It wouldn't make sense for them to each have implementations that do DB writes, or create CSVs. The only function of a processor should be to take in some kind of file blob, and return a vote object. That's what the `process_blob` method is meant to do. 

Adding methods that write to the database, means that that this class now needs to be concerned with the database connection details, the schema, etc. Same goes for CSV functionality. 

I think it makes more sense to handle the db writes in a separate class that is in charge of reading/writing processed vote records. 

Pretty soon we're going to want some methods that do things like 
`get all processed vote records in year x`, `get all of the votes in which the representative ${rep_name} voted no`, etc. Those methods aren't going to live in the processor class. They should probably live in a `ProcessedVoteRecordDao` class.

Reply:
I think we'd want to just return the parsed vote object [here](https://github.com/LoveShack-Inc/represent/blob/master/rep/main.py#L85), and then write it and mark it processed with the `ProcessedVoteObjectDao` or whatever you want to call it

Reply:
I agree, I'll make another class and see how that looks